AdvectionDiffusion.C

```diff
@@ -17,6 +17,10 @@
 #include "Utilities.h"
 #include "Vec3Oper.h"
 #include "AnaSol.h"
+#include "EqualOrderOperators.h"
+
+using WeakOps = EqualOrderOperators::Weak; //!< Convenience renaming
+using ResidualOps = EqualOrderOperators::Residual; //!< Convenience renaming
```

Comment:
OK, they where not used in the header, so therefore I moved them to file scope. But if you rely on them in other solvers using this we move them back, no big deal.